tests/container/Makefile

```diff
@@ -0,0 +1,28 @@
+IMAGE_TAG_BASE=ansible/receptor:base
+IMAGE_TAG_TEST=ansible/receptor:test
+
+image: image-base
+
+image-base:
+	podman build -t $(IMAGE_TAG_BASE) -f ./base/Dockerfile ../..
```

Comment:
s/Dockerfile/Containerfile/g


in the interest of being consistent with our other projects

Reply:
Done ^^